lib/vjs-hls.js

```diff
@@ -30,7 +31,49 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
         }
 
         function _onError(event, data) {
-            console.error('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details);
+            var error = {
+              message: ('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details),
+            };
+            console.error(error.message);
+
+            // increment/set error count
+            errorCounts[data.type] ? errorCounts[data.type] += 1 : errorCounts[data.type] = 1;
```

Comment:
Since we are not using grunt I just took the source code from github for our project and removed the require statements.  It would be great if a dist version of the project could be checked in into the repo or published to npm.

So what is different about the build version of this vs. the source version?